### ISSUE TITLE
[pgadmin4] Additional env variables setup

### ISSUE THREAD

**Issue by krezovic** [rowanruseler/helm-charts#9]
Hi,

It would be nice if we could have a better control of env variables. Currently, few options have explicit configuration, but for each new option we have to crete a new helm value, which is tiresome. Right now, I'd like to set two additional env vars:

```
            - name: PGADMIN_CONFIG_ALLOW_SAVE_PASSWORD
              value: 'False'
            - name: PGADMIN_CONFIG_UPGRADE_CHECK_ENABLED
              value: 'False'
```

and I'd either have to create two additional HELM options, or  extend usage of "values.env" with to generate the block similar to ingress.hosts. See also

https://github.com/helm/charts/blob/f2fcadaa5a1af57bbdf2e49a99f25c9589d5b540/stable/fluentd/templates/deployment.yaml#L68-L71

**Comment by rowanruseler** [comment]
@krezovic apologies for the late reply, but I think what your best approach is, would be to use the values.env as default values but add an own file above that with only the adjustment you want to have.

e.g.:

helm install --name pgadmin-c001 -f pgadmin-c001.yaml -f values.yaml runix/pgadmin4

At least, this is how I maintain it for multiple clusters.

**Comment by krezovic** [comment]
Hi,

I'm talking about ENV variables passed to the pgadmin4 docker container. I still have no way even with your approach to set the two variables above without directly modifying the templates/deployment.yaml.

TL;DR; no way to pass any variables to pod's env: array, only the ones you chose (there are at least two of them, I'd like to have additional two at the moment)
